include/revng/Pipes/RevngPipes.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
```

Comment:
Please split everything in here to its own header and call this one `AllPipes.h`.